test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -163,16 +176,20 @@ protected virtual string GetAdoInvariant()
             Assert.Equal(1, data.Members.Count);
 
             data = await membershipTable.ReadRow(newEntry.SiloAddress);
-            Assert.Equal(newTableVersion.Version, data.Version.Version);
+            if (extendedProtocol)
+                Assert.Equal(newTableVersion.Version, data.Version.Version);
 
             logger.Info("Membership.ReadRow returned VableVersion={0} Data={1}", data.Version, data);
 
             Assert.Equal(1, data.Members.Count);
 
-            Assert.NotNull(data.Version.VersionEtag);
-            Assert.NotEqual(newTableVersion.VersionEtag, data.Version.VersionEtag);
-            Assert.Equal(newTableVersion.Version, data.Version.Version);
+            if (extendedProtocol)
+            {
+                Assert.NotNull(data.Version.VersionEtag);
```

Comment:
I think so too. @galvesribeiro, please make another pass at the changes you made to the tests.